<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>R. Reucher</author>
 <class>MESSSoftwareList</class>
 <widget class="QWidget" name="MESSSoftwareList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MESS software list</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButtonReload">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Reload all information</string>
     </property>
     <property name="statusTip">
      <string>Reload all information</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/reload.png</normaloff>:/data/img/reload.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxDeviceConfiguration">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select a pre-defined device configuration to be added to the software setup</string>
     </property>
     <property name="statusTip">
      <string>Select a pre-defined device configuration to be added to the software setup</string>
     </property>
     <item>
      <property name="text">
       <string>No additional devices</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonAddToFavorites">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Add the currently selected software and device setup to the favorites list</string>
     </property>
     <property name="statusTip">
      <string>Add the currently selected software and device setup to the favorites list</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/plus.png</normaloff>:/data/img/plus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButtonRemoveFromFavorites">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Remove the currently selected favorite setup</string>
     </property>
     <property name="statusTip">
      <string>Remove the currently selected favorite setup</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/minus.png</normaloff>:/data/img/minus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="toolButtonPlay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Play the selected software configuration</string>
     </property>
     <property name="statusTip">
      <string>Play the selected software configuration</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/launch.png</normaloff>:/data/img/launch.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="toolButtonPlayEmbedded">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Play the selected software configuration (embedded)</string>
     </property>
     <property name="statusTip">
      <string>Play the selected software configuration (embedded)</string>
     </property>
     <property name="icon">
      <iconset resource="qmc2.qrc">
       <normaloff>:/data/img/embed.png</normaloff>:/data/img/embed.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QToolBox" name="toolBoxSoftwareList">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageKnownSoftware">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>305</width>
        <height>192</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="qmc2.qrc">
        <normaloff>:/data/img/flat.png</normaloff>:/data/img/flat.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Known software</string>
      </attribute>
      <attribute name="toolTip">
       <string>Complete list of known software for the current machine</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidgetKnownSoftware">
         <property name="toolTip">
          <string>List of known software for the current machine</string>
         </property>
         <property name="statusTip">
          <string>List of known software for the current machine</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Publisher</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>List</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFavoriteSoftware">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>383</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="qmc2.qrc">
        <normaloff>:/data/img/favorites.png</normaloff>:/data/img/favorites.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Favorites</string>
      </attribute>
      <attribute name="toolTip">
       <string>View / manage your favorite software list for the current machine</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidgetFavoriteSoftware">
         <property name="toolTip">
          <string>Favorite software for the current machine</string>
         </property>
         <property name="statusTip">
          <string>Favorite software for the current machine</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Publisher</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>List</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Device configuration</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSearchSoftware">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>575</width>
        <height>383</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="qmc2.qrc">
        <normaloff>:/data/img/hint.png</normaloff>:/data/img/hint.png</iconset>
      </attribute>
      <attribute name="label">
       <string>Search</string>
      </attribute>
      <attribute name="toolTip">
       <string>Search within the list of known software for the current machine</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxSearch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Search for known software (not case-sensitive)</string>
         </property>
         <property name="statusTip">
          <string>Search for known software (not case-sensitive)</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="treeWidgetSearchResults">
         <property name="toolTip">
          <string>Search results</string>
         </property>
         <property name="statusTip">
          <string>Search results</string>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Publisher</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Year</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>List</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <tabstops>
  <tabstop>toolButtonReload</tabstop>
  <tabstop>comboBoxDeviceConfiguration</tabstop>
  <tabstop>toolButtonAddToFavorites</tabstop>
  <tabstop>toolButtonRemoveFromFavorites</tabstop>
  <tabstop>toolButtonPlay</tabstop>
  <tabstop>toolButtonPlayEmbedded</tabstop>
  <tabstop>treeWidgetKnownSoftware</tabstop>
  <tabstop>treeWidgetFavoriteSoftware</tabstop>
  <tabstop>comboBoxSearch</tabstop>
  <tabstop>treeWidgetSearchResults</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
